fvm/meter/computation_meter.go

```diff
@@ -116,10 +116,24 @@ func (m *ComputationMeter) ComputationAvailable(
 	if !ok {
 		return true
 	}
+
 	potentialComputationUsage := m.computationUsed + w*uint64(intensity)
 	return potentialComputationUsage <= m.params.computationLimit
 }
 
+// ComputationRemaining returns the remaining computation left in the transaction for the given type
+func (m *ComputationMeter) ComputationRemaining(kind common.ComputationKind) uint {
+	w, ok := m.params.computationWeights[kind]
+	// if not found return has capacity
+	// given the behaviour of MeterComputation is ignoring intensities without a set weight
+	if !ok {
+		return math.MaxUint
+	}
+
+	remainingComputationUsage := m.params.computationLimit - m.computationUsed
```

Comment:
I think we should not rely on computation limit to be `>=` to computation used here.

Reply:
Good point! Added handling for this edge case in f075976943eb85ee0b111e4c89285bb45367af99